source/frontend/src/features/mapSideBar/tabs/HeaderField.tsx

```diff
@@ -31,8 +35,18 @@ export const HeaderLabelCol: React.FC<IHeaderLabelColProps> = props => (
 
 export const HeaderContentCol: React.FC<
   React.PropsWithChildren<IHeaderContentColProps>
-> = props => (
-  <Col xs={props.contentWidth ?? 'auto'} className="pl-1 text-left">
-    <strong>{props.children}</strong>
-  </Col>
-);
+> = props => {
+  return props.valueTestId ? (
```

Comment:
I don't love this, could we just set the datatestid to the prop value or empty string so we don't need a ternary?

Reply:
Removed ternary, React indeed doesn't add the property when the value passed is null or undefined.